critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
```

Comment:
There's less than a week left for the official GSoC coding period, so I'd suggest to avoid doing very time consuming tasks such as evaluating different libraries. If this one works well enough then the only thing we need to do is clean up the code to meet our requirements and move on.